x-pack/plugins/ml/public/jobs/new_job/simple/components/watcher/create_watch_view.js

```diff
@@ -0,0 +1,208 @@
+      });
+  }
+
+  onThresholdChange = (threshold) => {
+    this.setState({ threshold }, () => {
+      this.config.threshold = threshold;
+    });
+  }
+
+  onIntervalChange = (e) => {
+    const interval = e.target.value;
+    this.setState({ interval }, () => {
+      this.config.interval = interval;
+    });
+  }
+
+  onIncludeEmailChanged = (e) => {
+    const includeEmail = e.target.checked;
+    this.setState({ includeEmail }, () => {
+      this.config.includeEmail = includeEmail;
+    });
+  }
+
+  onEmailChange = (e) => {
+    const email = e.target.value;
+    this.setState({ email }, () => {
+      this.config.email = email;
+    });
+  }
+
+  render() {
+    const mlSelectSeverityService = {
+      state: {
+        set: (name, threshold) => {
+          this.onThresholdChange(threshold);
+          return {
+            changed: () => {}
+          };
+        },
+        get: () => {
+          return this.config.threshold;
+        },
+      }
+    };
+    const { status } = this.state;
+
+    if (status === null || status === STATUS.SAVING || status === STATUS.SAVE_FAILED) {
+      return (
+        <div className="create-watch">
+          <div className="form-group form-group-flex">
+            <div className="sub-form-group">
+              <div>
+                <label
+                  htmlFor="selectInterval"
+                  className="euiFormLabel"
+                >
+                  Time range
+                </label>
+              </div>
+              Now - <EuiFieldText
+                value={this.state.interval}
+                onChange={this.onIntervalChange}
+                aria-label="Use aria labels when no actual label is in use"
+              />
+            </div>
+
+            <div className="sub-form-group">
+              <div>
+                <label
+                  htmlFor="selectSeverity"
+                  className="euiFormLabel"
+                >
+                Severity threshold
+                </label>
+              </div>
+              <div className="dropdown-group">
+                <SelectSeverity
+                  mlSelectSeverityService={mlSelectSeverityService}
+                />
+              </div>
+            </div>
+          </div>
+          {
+            this.state.emailEnabled &&
+
+            <div className="form-group">
+              <EuiCheckbox
+                id="includeEmail"
+                label="Send email"
+                checked={this.state.includeEmail}
+                onChange={this.onIncludeEmailChanged}
+              />
+              {
+                this.state.includeEmail &&
+                <div className="email-section">
+                  <EuiFieldText
+                    value={this.state.email}
+                    onChange={this.onEmailChange}
+                    placeholder="email address"
+                    aria-label="Use aria labels when no actual label is in use"
```

Comment:
As above, this `aria-label` text looks like a copy and paste from the docs. It should say something like 'email address' I think